crates/dojo-core/src/world.cairo

```diff
@@ -19,16 +19,67 @@ trait IWorld<T> {
     fn entity(
         self: @T, model: felt252, keys: Span<felt252>, layout: dojo::database::introspect::Layout
     ) -> Span<felt252>;
+    fn entity_by_id(
+        self: @T, model_id: felt252, entity_id: felt252, layout: dojo::database::introspect::Layout
+    ) -> Span<felt252>;
     fn set_entity(
         ref self: T,
         model: felt252,
         keys: Span<felt252>,
         values: Span<felt252>,
         layout: dojo::database::introspect::Layout
     );
+    fn set_entity_by_id(
+        ref self: T,
+        model_id: felt252,
+        entity_id: felt252,
+        values: Span<felt252>,
+        layout: dojo::database::introspect::Layout
+    );
     fn delete_entity(
         ref self: T, model: felt252, keys: Span<felt252>, layout: dojo::database::introspect::Layout
     );
+    fn delete_entity_by_id(
+        ref self: T,
+        model_id: felt252,
+        entity_id: felt252,
+        layout: dojo::database::introspect::Layout
+    );
+
+    fn entity_member(
+        self: @T,
+        model_id: felt252,
+        keys: Span<felt252>,
+        member_id: felt252,
```

Comment:
If we keep the nomenclature, isn't `member_selector` here instead? Well.. actually it will depend on my other comment on the selector. :)